app/src/main/java/de/westnordost/streetcomplete/quests/postbox_royal_cypher/PostBoxRoyalCypher.kt

```diff
@@ -0,0 +1,12 @@
+package de.westnordost.streetcomplete.quests.postbox_royal_cypher
+
+enum class PostboxRoyalCypher(val osmValue: String) {
+    VICTORIA("VR"),
+    EDWARDVII("EVIIR"),
+    GEORGEV("GR"),
+    EDWARDVIII("EVIIIR"),
+    GEORGEVI("GVIR"),
+    ELIZABETHII("EIIR"),
+    SCOTTISHCROWN("scottish_crown"),
+    NONE("no")
```

Comment:
Since the naming of these constants is not linked to the OSM values, may I suggest to add underscores to make them more readable?

```suggestion
    VICTORIA("VR"),
    EDWARD_VII("EVIIR"),
    GEORGE_V("GR"),
    EDWARD_VIII("EVIIIR"),
    GEORGE_VI("GVIR"),
    ELIZABETH_II("EIIR"),
    SCOTTISH_CROWN("scottish_crown"),
    NONE("no")
```